### PR TITLE
A11y: announce copy to clipboard action

### PR DESCRIPTION
**Pull request by anurbol**
Screen reader did not announce any information upon activating 'Copy to Clipboard' button, with this change it will.

Made as per this [great guiedline](https://github.com/github/thehub/blob/main/docs/epd/engineering/dev-practicals/frontend/accessibility/readiness-routine/screenreaders/live-regions-and-screen-reader-announcements.md).

## References
Related to https://github.com/github/accessibility-audits/issues/2795

### CODE REVIEW

**Review by Seryozha95 — APPROVED** [review]

File: theme/src/aria-live.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+ function setContainerContent(message) {
+   if (!container) return
+   if (container.textContent === message) {
+     /* This is a hack due to the way the aria live API works.
+     A screen reader will not read a live region again
+     if the text is the same. Adding a space character tells
+     the browser that the live region has updated,
+     which will cause it to read again, but with no audible difference. */
+     container.textContent = `${message}\u00A0`
+
```
Comment (amit-avit) [inline theme/src/aria-live.js:45]: nit:

```suggestion
```

File: theme/src/aria-live.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   container.style.clip = 'rect(0, 0, 0, 0)'
+   container.style.wordWrap = 'normal'
+   container.style.border = '0'
+   document.body.append(container)
+ }
+ 
+ module.exports = {
+   announce
+ }
+
```
Comment (amit-avit) [inline theme/src/aria-live.js:69]: nit: remove whitespace

**Review by amit-avit — APPROVED** [review]
LGTM

File: theme/src/components/clipboard-copy.js [hunk 3]
```diff
@@ -27,6 +28,7 @@ function ClipboardCopy({value}) {
       onClick={() => {
         copy(value)
         setCopied(true)
+        announce(`copied to clipboard`)
```
Comment (amit-avit) [inline theme/src/components/clipboard-copy.js:31]: nit:

```suggestion
        announce(`Copied to clipboard`)
```

**Pull request merged**